### PR TITLE
[BD-26] [EDUCATOR-5758] Create API endpoint to provide Proctoring settings

### PR DESCRIPTION
**Pull request by idegtiarov**
**Description:**

Create an API endpoint to provide proctoring generic and backend specific instructions for the special exams
in the MFE app-learning.

**JIRA:**

[EDUCATOR-5758](https://openedx.atlassian.net/browse/EDUCATOR-5758)

**Pre-Merge Checklist:**

- [x] Updated the version number in `edx_proctoring/__init__.py` and `package.json` if these changes are to be released.
- [x] Described your changes in `CHANGELOG.rst`
- [x] Confirmed Github reports all automated tests/checks are passing.
- [x] Approved by at least one additional reviewer.

**Post-Merge:**

- [ ] Create a tag matching the new version number.

### CODE REVIEW

File: edx_proctoring/urls.py [hunk 1]
```diff
@@ -116,6 +116,11 @@
         views.ProctoredExamAttemptView.as_view(),
         name='proctored_exam.exam_attempts'
     ),
+    url(
+        r'edx_proctoring/v1/settings/exam_id/(?P<exam_id>\d+)/$',
```
Comment (Zacharis278) [inline edx_proctoring/urls.py:120]: sorry for a last minute comment. Looking at all the other endpoints we called during an exam the URL path is prefixed with `proctored_exam` can we be more closely copy these endpoints? Something like this:
```suggestion
        r'edx_proctoring/v1/proctored_exam/settings/exam_id/(?P<exam_id>\d+)/$',
```

**Comment by Zacharis278** [comment]
tagging @edx/masters-devs-cosmonauts for visibility

File: edx_proctoring/urls.py (hunk 1 above)
Comment (idegtiarov) [reply edx_proctoring/urls.py:120]: Thanks, I like your suggestion! Done.

**Review by Zacharis278 — APPROVED** [review]

**Comment by Zacharis278** [comment]
@idegtiarov I was ready to merge this but we lost the race to get our version number in. Can you update that? I can merge right away once the conflict is fixed

**Comment by Zacharis278** [comment]
@idegtiarov I think you'll need to merge the latest master down into this branch to resolve the conflict

**Comment by idegtiarov** [comment]
@Zacharis278 That's right, just did it! ))

**Pull request merged**